### PR TITLE
Upgrading home page

### PR DESCRIPTION
**Pull request by Vic-ST**
Here is an updated version. @GoRedbirds what do you think?

---
This change is [Reviewable](https://reviewable.io/reviews/stevencopeland/stores/43)

### CODE REVIEW

**Comment by Vic-ST** [comment]
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/43)*

**Comment by ruu5cn** [comment]
Could you put this on codecademy?

**Comment by Vic-ST** [comment]
Okay!

**Comment by Vic-ST** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[begin_round_four.js, line 38 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2i-r1-38:-Kg1Q_C8682LsrAAx5cT:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/begin_round_four.js#L38)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['other' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=576580)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/43)*

**Comment by Vic-ST** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[begin_round_for_3_players.js, line 67 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2h-r1-67:-Kg1QrkIpPSsmiZ7x0ot:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/begin_round_for_3_players.js#L67)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['other' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=576580)
</blockquote></details>

Done.

---

*[begin_round_number_two.js, line 69 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2j-r1-69:-Kg1Qo0zYmJu3Ld3oXWV:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/begin_round_number_two.js#L69)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: ['other' is not defined. (no-undef)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=576580)
</blockquote></details>

Done.

---

*[JavaScript-Files/jQuery.js, line 4 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2f-r1-4:-Kg1QpqofIz1ExIFiua3:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/JavaScript-Files/jQuery.js#L4)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: [The function hide can be unsafe (scanjs-rules/call_hide)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=576580)
</blockquote></details>

Done.

---

*[JavaScript-Files/jQuery.js, line 6 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2f-r1-6:-Kg1Qqnduwb-fgCmaGrn:b-896fix) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/JavaScript-Files/jQuery.js#L6)):*
<details><summary><i>Previously, codacy-bot (Codacy Bot) wrote…</i></summary><blockquote>

Codacy Issue found: [The function hide can be unsafe (scanjs-rules/call_hide)](https://www.codacy.com/app/StevenCopeland/Stores/pullRequest?prid=576580)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/43)*

File: JavaScript-Files/jQuery.js [hunk 1]
```diff
@@ -1,26 +1,55 @@
 $(document).ready(function() {
     $(".rules").hide();
     $(".game").hide();
+    $(".2playerfunc").hide();
```
[the remark this answers is not in the document]
Comment (Vic-ST) [reply JavaScript-Files/jQuery.js:4]: Very True!

**Comment by Vic-ST** [comment]
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[begin_round_four.js, line 38 at r1](https://reviewable.io:443/reviews/stevencopeland/stores/43#-Kg0L_w0SArWtxB57I2i-r1-38:-Kg1R45vNfNKv4m9AIDv:bo688cr) ([raw file](https://github.com/stevencopeland/stores/blob/556afda84952ea03dcca721677b7076ed508d20d/begin_round_four.js#L38)):*
> one

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/43)*

**Comment by Vic-ST** [comment]
Reviewed 1 of 6 files at r1.
Review status: all files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/43)*

File: begin_round_number_two.js [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                var max = Math.floor(person.money/20);
+                var askP = prompt("How many workers would you like to buy? You have " + person.money + " dollars. The most you can buy is " + max, max)*1;
+                if (person.money - (askP * 20) < 0) {
+                    confirm("Oops! Sorry, it looks like you don't have enough money.");
+                    ask();
+                }else if(isNaN(askP)) {
+                    alert("Sorry! That is not a number!");
+                    ask();
+                } else {
+                    var cost = askP*20;
```
[the remark this answers is not in the document]
Comment (Vic-ST) [reply begin_round_number_two.js:69]: Not sure what this one means

File: begin_round_for_3_players.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+        var personPersonBuyWorkers = function() {
+            var ask = function() {
+                var max = Math.floor(person.money/20);
+                var askP = prompt("How many workers would you like to buy? You have " + person.money + " dollars. The most you can buy is " + max, max)*1;
+                if (person.money - (askP * 20) < 0) {
+                    confirm("Oops! Sorry, it looks like you don't have enough money.");
+                    ask();
+                }else if(isNaN(askP)) {
+                    alert("Sorry! That is not a number!");
+                    ask();
```
[the remark this answers is not in the document]
Comment (Vic-ST) [reply begin_round_for_3_players.js:67]: Or this one.

**Comment by Vic-ST** [comment]
@GoRedbirds I am closing this and deleting this branch.

**Pull request closed without merging**